src/components/UNSTABLE__Shell/HeaderActions/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file Header actions.
+ * @copyright IBM Security 2020
+ */
+
+import { node } from 'prop-types';
+import React, { Children, cloneElement, useState } from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as shellNamespace } from '../Shell';
+import { headerNamespace } from '../ShellHeader';
+
+const suffix = 'group';
+
+const namespace = appendComponentNamespace(shellNamespace, suffix);
+
+function HeaderActions({ children, ...other }) {
+  const [activeAction, setActiveAction] = useState(null);
+
+  return (
+    <div
+      className={`${appendComponentNamespace(
+        headerNamespace,
+        suffix
+      )} ${namespace}`}
+      {...other}
+    >
+      {Children.map(children, (child, index) =>
+        cloneElement(child, {
```

Comment:
Could we avoid `cloneElement` for easier debugging?

Reply:
Resolving thread in favor of https://github.com/carbon-design-system/ibm-security/pull/285#discussion_r384737204